tests/phpunit/CRM/Contribute/Form/ContributionTest.php

```diff
@@ -1260,6 +1260,47 @@ public function testReSubmitSaleTaxAlteredAmount(string $thousandSeparator): voi
     $this->assertEquals(1000, $items['values'][3]['amount']);
   }
 
+  /**
+   * Test Changing FT on Quick Config Price Set
+   */
+  public function testSubmitFormChangeFTQuickConfig(): void {
+    $this->submitContributionForm([
+      'total_amount' => $this->formatMoneyInput(10000),
+      'financial_type_id' => $this->financialTypeID,
+      'receive_date' => '2015-04-21 00:00:00',
+      'contact_id' => $this->ids['Contact'][0],
+      'payment_instrument_id' => $this->getPaymentInstrumentID('Check'),
+      'contribution_status_id' => 1,
+    ]);
+    $contribution = $this->callAPISuccessGetSingle('Contribution',
+      [
+        'contribution_id' => 1,
+      ],
+    );
+    $financialTypeID = CRM_Core_DAO::getFieldValue('CRM_Financial_DAO_FinancialType', 'Campaign Contribution', 'id', 'name');
+    $this->submitContributionForm([
+      'id' => $contribution['id'],
+      'total_amount' => $this->formatMoneyInput(10000),
+      'financial_type_id' => $financialTypeID,
+      'receive_date' => $contribution['receive_date'],
+      'payment_instrument_id' => $contribution['payment_instrument_id'],
+      'price_set_id' => 0,
+      'check_number' => 12345,
```

Comment:
This is a bit odd for an edit since there is no such field on the form on edit, but probably not relevant to what is being tested.